curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6555dd138e70cae6b546966d.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-58
 
 # --description--
 
-You need to get the currently playing song or the first song in the playlist. To do that, create a `song` constant and set it to the current song of `userData`, **or** the first song in the `userData?.songs` array.
+You need to get the currently playing song or the first song in the playlist. To do that, create a `song` constant and use the OR operator ( || ) to set it to the current song of `userData`, **or** the first song in the `userData?.songs` array.
```

Comment:
```suggestion
You need to get the currently playing song or the first song in the playlist. To do that, create a `song` constant and use the OR operator (`||`) to set it to the current song of `userData`, **or** the first song in the `userData?.songs` array.
```
With backticks it might be more clear.

Reply:
I will make the change